addons/binding/org.openhab.binding.vitotronic/ESH-INF/binding/binding.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <binding:binding id="vitotronic"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
-        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+                 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
I followed your hint to use the IDE and formatted all *.xml and *.java
Let´s see what is the outcome. This time the travis build failed for other reasons and I can´t check if the reformatted files were o.k.
@kaikreuzer will the travis build start again automatically when the problems are fixed or do I have to do something?